### ISSUE TITLE
Page slowing on http://blackhawks.nhl.com/club/page.htm?id=106807

### ISSUE THREAD

**Issue by subwaymatch** [subwaymatch/trackbrowser-client#1]
Hobson, Jessen L
3:32 AM (4 hours ago)

to me 
Significant slowing on http://blackhawks.nhl.com/club/page.htm?id=106807
The screen would shrink and then go back to normal size and then repeat that. Did work eventually (30 seconds)

**Comment by subwaymatch** [comment]
Multiple events for "did-frame-finish-load" are being fired in certain pages including this page. As a result, screenshots are taken simultaneously for each "did-frame-finish-load" event. 

Need to see if there are ways to check for duplicate "did-frame-finish-load" events on certain pages

logs

```
1/31/2016, 2:56:03 AM
parktest
navigation
User navigated to http://blackhawks.nhl.com/club/page.htm?id=106807.

1/31/2016, 2:56:03 AM
parktest
navigation
User navigated to http://blackhawks.nhl.com/club/page.htm?id=106807.

1/31/2016, 2:56:01 AM
parktest
navigation
User navigated to http://blackhawks.nhl.com/club/page.htm?id=106807.
```

**Comment by subwaymatch** [comment]
Issue fixed: 60eff48f05ae90099c69ed770831ba5d33d9e80d
